modules/persistent-storage-csi-ebs-operator-install.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+
+[id="persistent-storage-csi-ebs-operator-install_{context}"]
+= Installing the AWS Elastic Block Store CSI Operator
+
+The AWS Elastic Block Store (EBS) Container Storage Interface (CSI) Operator enables the replacement of the existing AWS EBS in-tree storage plug-in.
+
+When you install the AWS EBS CSI Operator, {product-title} provides the CSI driver that allows you to use CSI volumes with the `PersistentVolumeClaims`, `PersistentVolumes`, and `StorageClasses` API objects.
+
+The AWS EBS CSI Operator is not installed in {product-title} by default. Use the following procedure to install and configure this Operator to enable the CSI AWS EBS driver in your cluster.
+
```

Comment:
To consistent with others, it is better to update to "enable the AWS EBS CSI driver"

Reply:
good catch, fixed